src/IdePurescript/Atom/Main.purs

```diff
@@ -265,6 +266,23 @@ main = do
             liftEff $ setTextInBufferRange ed range $ intercalate "\n" lines
         Nothing -> pure unit
 
+    fixTypo :: Eff MainEff Unit
+    fixTypo = do
+      runAff raiseError ignoreError $ runMaybeT body
+      where
+      body :: MaybeT (Aff MainEff) Unit
+      body = do
+        ed :: TextEditor <- MaybeT $ liftEff $ getActiveTextEditor atom.workspace
+        { line, col, pos, range } <- lift $ liftEff $ getLinePosition ed
+        { word, range: wordRange } <- MaybeT $ liftEff $ getToken ed pos
+        corrections <- lift $ eitherToErr (P.suggestTypos word 2)
+        liftEff $ selectListViewStatic view (replaceTypo ed wordRange) Nothing (runCompletion <$> corrections)
+        where
+          runCompletion (Completion obj) = obj
+          replaceTypo ed wordRange { identifier } = void $ unsafeInterleaveEff (setTextInBufferRange ed wordRange identifier)
+          view {identifier, "module'": m} = "<li>" ++ m ++ "." ++ identifier ++ "</li>"
```

Comment:
not sure you want to see a qualified name here? Admittedly it might make more sense (and for the other place this appears) with some styling